packages/databricks-vscode/resources/whats-new/1.2.md

```diff
@@ -0,0 +1,48 @@
+# <!-- DATABRICKS LOGO --> Databricks extension for Visual Studio Code v1.2.0
+
+## What's new?
+
+-   [📗 Advanced notebook support with Databricks Connect](#dbconnect-notebook)
+
+## <a id="dbconnect-notebook"></a> Advanced notebook support with Databricks Connect
+
+Before this update, you could already run notebooks locally with Databricks Connect. This update introduces advanced notebook support. This advanced support enables you to run notebooks locally with a similar experience as you have by running them on a Databricks cluster. The goal of this update is to bring much of the convenience of running notebooks on a Databricks cluster to your local development environment.
+
+This update includes:
+
+-   [Magic commands such as `%run` and `%sql`](#magic-commands)
+-   [Preconfigured globals such as `spark` and `dbutils`](#preconf-globals)
+-   [Interactive widgets through `dbutils.widgets`](#widgets)
+-   [Support for Databricks notebooks](#dbnb)
+
+### <a id="magic-command"></a>Magic commands
+
+You can now use Databricks magic commands locally from your notebooks.
+
+For a list of supported magic commands, see [link-text](url).
+
+![magic_sql](./1.2/magic_sql.gif)
+
+### <a id="preconf-globals"></a>Preconfigured globals
+
+Just like on a Databricks cluster, this update configures and provides you with some globals such as `spark` and `dbutils` when you run notebooks locally.
+
+For a list of supported globals, see [link-text](url).
```

Comment:
I would leave this out, or link to the upcoming article in the tech docs.

Reply:
@PaulCornellDB I will wait for the docs to publish before we send this out. I will definitely want deep links everywhere.